hvcc/generators/c2fmod/templates/src/{{name}}.plugin.js

```diff
@@ -0,0 +1,11 @@
+studio.plugins.registerPluginDescription("{{name}}", {
+  companyName: "PlugData",
```

Comment:
why use `PlugData` as the default here?

would you mind `Wasted Audio` instead? ;)